scripts/apps/authoring/authoring/directives/AuthoringTopbarDirective.ts

```diff
@@ -45,13 +46,33 @@ export function AuthoringTopbarDirective(
             };
 
             scope.openPublishOrSendToPane = () => {
+                const activeTab = determineActiveTab(scope.item);
+
                 dispatchInternalEvent('interactiveArticleActionStart', {
                     items: [scope.item],
-                    tabs: ['send_to', 'publish'],
-                    activeTab: scope.item.flags?.marked_for_not_publication ? 'send_to' : 'publish',
+                    tabs: getTabsArray(activeTab),
+                    activeTab: activeTab,
                 });
             };
 
+            function determineActiveTab(item) {
```

Comment:
This is making implicit assumptions about which tabs are available. First check if a tab exists in the list of available tabs and add a fallback to 1st tab.

Reply:
available tabs are decided based on active tab:
if `activeTab = correct` so in this case `availableTabs = ['send_to', 'correct']` and if `activeTab = publish/send_to` in this case, `availableTabs = ['send_to', 'publish']`
So first, we have to check activeTab based on the item. then send both activeTab and availableTabs